ballerina/Module.md

```diff
@@ -266,7 +266,7 @@ int id = 10;
 int age = 12;
 sql:ParameterizedQuery query = `SELECT * FROM students
                                 WHERE id < ${id} AND age > ${age}`;
-stream<Student, sql:Error> resultStream = dbClient->query(query);
+stream<Student, sql:Error?> resultStream = dbClient->query(query);
```

Comment:
Not sure why these changes appear here. All changes in Module.md were merged in https://github.com/ballerina-platform/module-ballerinax-mssql/pull/29/ and are currently present in the main branch https://github.com/ballerina-platform/module-ballerinax-mssql/blob/main/ballerina/Module.md